src/main/java/net/justonedev/kit/export/FileExporter.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the MIT License (MIT)
+ * Copyright (c) 2024 Benjamin Claus
+ */
+package net.justonedev.kit.export;
```

Comment:
Please move that to the namespace of the project. Also please use the same license header as in the other files. They are generated by spotless:apply. And therefore, they update based on the modifications.

Reply:
Done using spotless:apply in commit 6d27ac5d

Reply:
fine for me.